parts/k8s/cloud-init/masternodecustomdata.yml

```diff
@@ -243,6 +244,7 @@ write_files:
         user: client
       name: localclustercontext
     current-context: localclustercontext
+    #CLOUD_INIT_WAS_HERE
```

Comment:
will this work? I don't know if it would be interpreted in a comment since this is a kubeconfig file

Reply:
`#` as the first char in a line is a valid yaml comment